go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/osbuild/osbuild-composer
 go 1.12
 
 require (
+	github.com/aws/aws-sdk-go v1.25.37
```

Comment:
Any change in `go.mod` without corresponding change in the spec file will result in broken rpm build. I think we should add it as part of this PR even though we don't yet have RPM build as part of CI.

Reply:
Ah, good catch! Thanks, and I agree, we should update the spec file in the same patch. I was first thinking that this tool is not going to be shipped, so we should find a way of not requiring the dep in the rpm build, but the same dependency will anyway soon land in the worker, so that is not going to be worth the effort. I'll update.